File: 2729.check-if-the-number-is-fascinating.cs
/*
 * @lc app=leetcode id=2729 lang=csharp
 *
 * [2729] Check if The Number is Fascinating
 */

// @lc code=start
public class Solution {
    public bool IsFascinating(int n) {
        string resultString = "";
        for (int i = 1; i <= 3; i++) {
            resultString += (i * n).ToString();
        }
        for (int i = 1; i <= 9; i++) {
            if (resultString.Count(c => c == i.ToString()[0]) != 1) {
                return false;
            }
        }
        return true;
    }
}

// @lc code=end

File: 2729.check-if-the-number-is-fascinating.py
#
# @lc app=leetcode id=2729 lang=python3
#
# [2729] Check if The Number is Fascinating
#

# @lc code=start
class Solution:
    def isFascinating(self, n: int) -> bool:
        resultString = ""
        for i in range(1, 4, 1):
            resultString += str(i * n)
        for i in range(1, 10, 1):
            if resultString.count(str(i)) != 1:
                return False
        return True
        
# @lc code=end

